doc/source/Tutorials/GettingStarted/index.md

```diff
@@ -1,407 +1,499 @@
+    ![The first render](images/mainRenderGrey.png "The first render")
 
-Hopefully this provides a solid basis for your own further exploration, which will no doubt be less creatively challenged. You might like to start by exploring the addition of more lights, found in the _/Appleseed/Lights_ node menu, or by creating more expansive shading networks using the shaders found in the _/Appleseed/Shaders_ node menu.
+Congratulations! You have successfully rendered your first image. Gaffy is currently lacking shading, lighting, and texturing. We will move on to those soon. First, you should adjust the UI to provide yourself a more optimal workflow.
 
-[1]: ../../Installation/index.md
-[2]: ../../Reference/NodeReference/GafferScene/SceneReader.md
-[3]: ../../Reference/UIReference/GraphEditor.md
-[4]: ../../Reference/UIReference/Viewer.md
+
+## Pinning an Editor to a Node ##
+
+As mentioned earlier, the _Viewer_, _Scene Hierarchy_, and _Node Editor_ (each an **editor**) show their respective outputs of the currently selected node. This is not always convenient, because often you will need to edit one node while viewing the output of another. You can solve this by **pinning** an editor while a node is selected, which keeps that editor focused on the node.
+
+To make switching between viewing Gaffy's geometry and the render easier, you can modify the UI so you're working with two Viewers. First, start by pinning the last node in the graph that contains the scene:
+
+1. Select the InteractiveAppleseedRender node.
+
+2. Click ![the pin button](images/targetNodesUnlocked.png "Pin button") at the top-right
+  of the top pane. The pin button will highlight: ![highlighted pin](images/targetNodesLocked.png "Highlighted pin").
+
+The _Viewer_ is now locked to the InteractiveAppleseedRender node's scene (which contains all of the parts from its upstream scenes), and will only show that scene, even if you deselect it or select a different node.
+
+Next, pin the same node to the _Scene Hierarchy_. This time, use the middle-click shortcut:
+
+1. Select the InteractiveAppleseedRender node.
+
+2. Middle-click and drag the InteractiveAppleseedRender node from the _Graph Editor_ into the
+  _Scene Hierarchy_.
+
+As with the _Viewer_, the _Scene Hierarchy_ will now remain locked to the output of InteractiveAppleseedRender, regardless of your selection. Now you are free to select any node for editing in the _Node Editor_, but you will always be viewing the final results of the last node in the graph.
+
+For the final adjustment to the UI, create another _Viewer_ in the top-left panel, and pin the Catalogue node to it:
+
+1. At the top-right of the top panel, click ![the layout menu button](images/layoutButton.png "Layout menu button") to open the layout menu.
+
+2. Select _Viewer_. A new _Viewer_ will appear on the panel next to the first one.
+
+3. Middle-click and drag the Catalogue node onto the new _Viewer_. That _Viewer_ is now pinned to that node.
```

Comment:
Should we also explain pane splitting? Typically I'd want to see both viewers at the same time.